relay-server/src/services/store.rs

```diff
@@ -1290,6 +1289,8 @@ struct MetricKafkaMessage<'a> {
     timestamp: UnixTimestamp,
     tags: &'a BTreeMap<String, String>,
     retention_days: u16,
+    #[serde(default, skip_serializing_if = "Option::is_none")]
```

Comment:
```suggestion
    #[serde(skip_serializing_if = "Option::is_none")]
```

Default is implied for options